packages/gitlab/src/components/utils.tsx

```diff
@@ -128,3 +129,34 @@ export const parseGitLabReadme = (readme: string): string => {
 
     return modifiedLines.join('\n');
 };
+
+export const resolveReadmeRelativelinks = (
```

Comment:
I think we can do it better by using `remark`; in this way, the code is more readable:
For example  something like:
```ts
import { visit } from 'unist-util-visit'

export default function linksAbsolute (options) {
  return (tree, file) =>
    visit(tree, 'link', (node) => {
        node.url = ...
    })
  }
  ```


and then add the remark plugin here: https://github.com/immobiliare/backstage-plugin-gitlab/blob/df409a79cddc27e2b96460a1f8e62cd2669495b0/packages/gitlab/src/components/widgets/ReadmeCard/ReadmeCard.tsx#L144


What do you think about ?